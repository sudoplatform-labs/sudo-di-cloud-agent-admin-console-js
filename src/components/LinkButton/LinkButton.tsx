import styled from 'styled-components';

export const LinkButton = styled.button`
  display: inline;
  line-height: inherit;
  border: 0;
  outline: 0;
  color: currentColor;
  background: transparent;
  padding: 0;
  cursor: pointer;

  &:hover {
    text-decoration: underline;
  }
`;
