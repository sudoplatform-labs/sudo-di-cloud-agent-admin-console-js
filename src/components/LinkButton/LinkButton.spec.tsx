import { shallow } from 'enzyme';
import * as React from 'react';
import { LinkButton } from './LinkButton';

describe('LinkButton', () => {
  it('should render correctly', () => {
    const component = shallow(<LinkButton />);

    expect(component).toMatchSnapshot();
  });
});
