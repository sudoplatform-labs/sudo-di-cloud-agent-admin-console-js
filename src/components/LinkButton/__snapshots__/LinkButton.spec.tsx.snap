// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkButton should render correctly 1`] = `
.c0 {
  display: inline;
  line-height: inherit;
  border: 0;
  outline: 0;
  color: currentColor;
  background: transparent;
  padding: 0;
  cursor: pointer;
}

.c0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<button
  className="c0"
/>
`;
